inference-engine/thirdparty/clDNN/tests/test_cases/permute_gpu_test.cpp

```diff
@@ -836,3 +836,697 @@ TEST(permute_gpu_f32, basic_bfzyx_permute_0_2_3_4_1)
     }
 
 }
+
+/**
+ * Test cases for permute_tile_8x8 kernel
+ *
+ * This TCs are enabled only when batch axis move to the last.
+ * i.e permute order is 0,3,1,2 or 0,4,1,2,3 or 0,5,1,2,3,4
+ */
+TEST(permute_gpu_f32_tile_8x8, normal_bfyx_0_3_1_2) {
```

Comment:
fix names as 8x8 -> 8x8_4x4 becuase now it handles both tiling

Reply:
Fixed the comments as 8x8_4x4